wporg-gp-translation-events.php

```diff
@@ -26,7 +26,8 @@
 use WP_Query;
 
 class Translation_Events {
-	const CPT = 'translation_event';
+	const CPT      = 'translation_event';
+	const DB_TABLE = 'translate_event_actions';
```

Comment:
The `DB_TABLE` const seems to be unused.

```suggestion
```

Reply:
I looked for this `DB_TABLE` constant in the plugin, and it is unused, so I am going to approve this suggestion.